Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/DebugModalViewController.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  DebugModalViewController.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 1/30/23.
+//  Copyright © 2023 Appcues. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+// A UIViewController that can be used to host any SwiftUI modal content to be presented,
+// and handle keyboard avoidance, if necessary.
+//
+// This is similar to the structure of the ExperienceWrapperViewController - providing
+// the backdrop - and the ExperienceStepViewController centered within - providing the scroll
+// container where the hosted SwiftUI content resides. However, this is a more specific and
+// reduced implementation that is decoupled from Experience step content and data.
+@available(iOS 13.0, *)
+internal class DebugModalViewController<Content>: UIViewController where Content: View {
```

Comment:
Is this different from `internal class DebugModalViewController<Content: View>: UIViewController {`?

I'm more used to seeing a generic type constraint after the parameter name.

Reply:
I believe totally equivalent, will update to match `ExperienceStepRootView` to keep consistent, thanks 👍